ocaml/idl/ocaml_backend/ocaml_utils.ml

```diff
@@ -89,7 +89,7 @@ let rec ocaml_of_ty = function
   | Map(l, r) -> "("^alias_of_ty l^" * "^alias_of_ty r^") list"
   (*  | Ref "session" -> "Uuid.cookie" *)
   | Ref ty -> "[`"^ty^"] Ref.t"
-  | Option x -> failwith "ocaml_of_ty got an option"
+  | Option x -> ocaml_of_ty x ^ " option"
```

Comment:
This is a fixup for the previous commit introducing the option type handling in the datamodel? Could you add a note to its commit message that this should be squashed?